AppKit/CPView.j

```diff
@@ -246,6 +248,9 @@ var CPViewHighDPIDrawingEnabled = YES;
 
     id                  _animator;
     CPDictionary        _animationsDictionary;
+
+    // CPViewController support
+    CPViewController _viewController @accessors(property=_viewController);
```

Comment:
I'm not convinced by this mechanism. Would it be better to have an internal mechanism (notification for instance) to notify that a view will be removed, added or hidden ?